src/network/networkd-route.h

```diff
@@ -41,6 +41,8 @@ struct Route {
         unsigned char pref;
         unsigned flags;
 
+        bool dev;
```

Comment:
I think this bool should probably be called "on_device" too, if the configuration option is named like that now...